### ISSUE TITLE
add repository parameters to pypi

### ISSUE THREAD

**Issue by dhuckins** [getsentry/craft#24]
would it be possible to add additional parameters to the pypi upload?
specifically `TWINE_REPOSITORY_URL` for uploading to private/internal repositories

can submit PR

**Comment by jan-auer** [comment]
Definitely, PR appreciated. You can add it in a similar fashion as the credentials are set, or also add it to the target config.

https://github.com/getsentry/craft/blob/f06a0157989134eecf3127272f1e9fd2688b589e/src/targets/pypi.ts#L60-L63
